resources/dd-ix.net.zone

```diff
@@ -0,0 +1,60 @@
+; Check me with (abs. path required....)
+; named-checkzone dd-ix.net /.../resources/dd-ix.net.zone
+
+$TTL    1h         ; default TTL for zone
+$ORIGIN dd-ix.net. ; base domain-name
```

Comment:
I would drop the `$ORIGIN` as it is set to the zone name by default (this makes the zone also reusable)